app/javascript/sdk/IFrameHelper.js

```diff
@@ -39,6 +39,15 @@ export const IFrameHelper = {
       holderClassName += ` woot-widget--without-bubble`;
     }
     addClass(widgetHolder, holderClassName);
+
+    let closeBtn = document.createElement('button');
+    closeBtn.id = 'closeBtn';
```

Comment:
We might want a more scoped `id` for this, something that includes the name chatwoot maybe?